src/devices/machine/i8256.cpp

```diff
@@ -389,7 +435,7 @@ void i8256_device::write(offs_t offset, u8 data)
 			m_interrupts = m_interrupts & ~data;
 			break;
 		case I8256_REG_BUFFER:
-			LOG("I8256 write serial: %u\n", data);
+			LOG("I8256 write serial: %u %c\n", data, data);
```

Comment:
You want to be careful about this – you should sanitise the character before formatting with `%c`.